app/models/manageiq/providers/kubevirt/infra_manager/vm/operations/guest.rb

```diff
@@ -1,12 +1,14 @@
 module ManageIQ::Providers::Kubevirt::InfraManager::Vm::Operations::Guest
 
   def raw_reboot_guest
-    kubevirt = ext_management_system.parent_manager.connect(:service => "kubernetes", :path => "/apis/subresources.kubevirt.io", :version => "v1")
-    kubevirt.rest_client["namespaces/#{location}/virtualmachineinstances/#{name}/softreboot"].put({}.to_json, { 'Content-Type' => 'application/json' }.merge(kubevirt.get_headers))
```

Comment:
This was a good example of a method call that kubeclient couldn't handle, and `oc vm` can't do either and you have to use virtctl for.  (This was always gross but didn't want to block @Dhamo1107 from adding support for this while he was waiting for me to do this change).